tsl/src/import/ts_simplehash.h

```diff
@@ -0,0 +1,1069 @@
+/*
+ * This file and its contents are licensed under the Timescale License.
+ * Please see the included NOTICE for copyright information and
+ * LICENSE-TIMESCALE for a copy of the license.
+ */
```

Comment:
I know that Postgres license is very permissive, but doesn't make sense to vendor this code on Apache code?